_ingest-pipelines/processors/ml-inference.md

```diff
@@ -0,0 +1,158 @@
+| `model_config` | Object | Optional | Custom configuration options for the machine learning model.                                                                                                                                                                                                                                                                                                                                                      |
+| `max_prediction_tasks` | Int | Optional | The setting of the maximum number of prediction tasks can run when ingesting a document, default value is 10.                                                                                                                                                                                                                                                                                                     |
+| `description` | String | Optional | A brief description of the processor.                                                                                                                                                                                                                                                                                                                                                                             |
+| `tag` | String | Optional | An identifier tag for the processor. Useful for debugging to distinguish between processors of the same type.                                                                                                                                                                                                                                                                                                     |
+| `ignore_missing` | Boolean | Optional | If true and any of the input fields defined in input_map or output_map are missing, those missing fields are quietly ignored. Otherwise, a missing field causes a failure. Default is false.                                                                                                                                                                                                                      |
+| `ignore_failure` | Boolean | Optional | If true and any exceptions occur, they are quietly ignored, and the ingestion continues. Otherwise, an exception can occur and abort the ingestion. Default is false.                                                                                                                                                                                                                                             |
+
+## Using the processor
+
+Follow these steps to use the processor in a pipeline. You must provide a model ID when creating the processor. For more information, see [Using custom models within OpenSearch]({{site.url}}{{site.baseurl}}/ml-commons-plugin/using-ml-models/).
```

Comment:
We only support remote model in `ml_inference` in 2.14 release,  we should link to this doc https://opensearch.org/docs/latest/ml-commons-plugin/remote-models/index/

Reply:
added